src/chatClient.cpp

```diff
@@ -1852,6 +1855,7 @@ promise::Promise<void> GroupChatRoom::decryptTitle()
         wptr.throwIfDeleted();
         KR_LOG_ERROR("Error decrypting chat title for chat %s:\n%s\nFalling back to member names.", karere::Id(chatid()).toString().c_str(), err.what());
         makeTitleFromMemberNames();
+        return err;
```

Comment:
Then, @alber2510, you need to add the .fail() to handle the error properly in any outer function calling `decryptTitle()`, as Alex pointed above. If you have doubts, check with him.